python/lsst/cp/pipe/ptc/astierCovPtcUtils.py

```diff
@@ -233,36 +227,7 @@ def covDirectValue(diffImage, weightImage, dx, dy):
     return cov, nPix
 
 
-class LoadParams:
-    """
-    A class to prepare covariances for the PTC fit.
-
-    Parameters
-    ----------
-    r: `int`, optional
-        Maximum lag considered (e.g., to eliminate data beyond a separation "r": ignored in the fit).
-
-    subtractDistantValue: `bool`, optional
-        Subtract a background to the measured covariances (mandatory for HSC flat pairs)?
-
-    start: `int`, optional
-        Distance beyond which the subtractDistant model is fitted.
-
-    offsetDegree: `int`
-        Polynomial degree for the subtraction model.
-
-    Notes
-    -----
-    params = LoadParams(). "params" drives what happens in he fit. LoadParams provides default values.
-    """
-    def __init__(self):
-        self.r = 8
-        self.subtractDistantValue = False
-        self.start = 5
-        self.offsetDegree = 1
-
-
-def parseData(dataset, params):
+def parseData(dataset):
```

Comment:
This function just creates the `CovFit` objects that are later fit, correct?  I'm unhappy with it returning a dict but calling the variable name a list, but that's a side point.  I'm not entirely sure I see the point of generating a dictionary of `CovFit` objects, then iterating over the items in that dict instead of just doing a normal loop over amplifiers stored in the dataset, instantiating a `CovFit` object, running the fit, storing the useful fit values in the dataset.  Are the `CovFit` objects used downstream in a way that I'm just not thinking about correctly?

Reply:
Yes, the dictionary with the `CovFit` objects is then used in `cpSolvePtcTask` by `getOutputPtcDataCovAstier` when getting the output `dataset`.